### PR TITLE
PHP >= 7.1 compatible release

### PR DESCRIPTION
**Pull request by mabar**
Dropped compatibility with php 5.6 and 7.0
Added strict types
Latest symfony/console allowed
Dropped unuseful annotations (where were same as php native typehint)
~~Replaced DateTime with DateTimeInterface~~ - not possible, not supported by https://github.com/dragonmantank/cron-expression

### CODE REVIEW

**Comment by f3l1x** [comment]
Thanks. :-) I've released v0.1 with PHP 5.6, for our old systems and >=0.2 will be PHP 7.1 only.

**Comment by f3l1x** [comment]
Please, upgrade to latest ninjify/qa to complete this PR. :-) Good job btw.

File: composer.json [hunk 1]
```diff
@@ -16,15 +21,16 @@
...
     "dragonmantank/cron-expression": "^1.2.1",
     "nette/di": "^2.4.10",
-    "symfony/console": "^3.3.14"
+    "symfony/console": "^3.3.14 || ^4.0.0"
   },
   "require-dev": {
-    "ninjify/qa": "~0.4.0",
-    "mockery/mockery": "~1.0",
-    "phpunit/phpunit": "~5.7.26"
+    "ninjify/qa": "~0.8.0",
```
Comment (f3l1x) [inline composer.json]: `"ninjify/qa": "^0.8.0"`

File: composer.json [hunk 2]
```diff
@@ -16,15 +21,16 @@
...
+    "mockery/mockery": "~1.0.0",
+    "phpunit/phpunit": "~5.7.27",
+    "tracy/tracy": "2.4.9"
```
Comment (f3l1x) [inline composer.json]: `"tracy/tracy": "~2.4.14"`

File: src/Command/ListCommand.php [hunk 3]
```diff
@@ -18,49 +18,16 @@ class ListCommand extends Command
...
-	}
-
-	/**
-	 * @param string $key
-	 * @param IJob $job
-	 * @param DateTime $dateTime
 	 * @return string[]
 	 */
-	private static function formatRow($key, IJob $job, DateTime $dateTime)
+	private static function formatRow(string $key, IJob $job, DateTime $dateTime): array
```
Comment (f3l1x) [inline src/Command/ListCommand.php:49]: Move it to the end of the file.

File: src/Helpers/Debugger.php [hunk 4]
```diff
@@ -1,20 +1,17 @@
...
-	/**
-	 * @param Exception $e
-	 * @return void
-	 */
-	public static function log(Exception $e)
+	public static function log(Throwable $e): void
 	{
-		if (class_exists('\Tracy\Debugger')) {
-			\Tracy\Debugger::log($e);
+		if (class_exists(TracyDebugger::class)) {
```
Comment (f3l1x) [inline src/Helpers/Debugger.php:13]: Does it really works without the tracy/tracy package? I'm not sure. Have you tested it?

**Review by f3l1x — CHANGES_REQUESTED** [review]

File: src/Helpers/Debugger.php (hunk 4 above)
Comment (mabar) [reply src/Helpers/Debugger.php:13]: Yes, autoloading is not triggered by use statements, safe to use everywhere :)

Comment (mabar) [reply src/Helpers/Debugger.php:13]: http://sandbox.onlinephpfunctions.com/code/1d3d9fa8a8199e9bdabb60570cb73e87280bf412

Comment (f3l1x) [reply src/Helpers/Debugger.php:13]: Cool. thanks.

**Pull request merged**